components/profile-controller/controllers/profile_controller.go

```diff
@@ -588,3 +591,17 @@ func removeString(slice []string, s string) (result []string) {
 	}
 	return
 }
+
+func updateNamespaceLabels(ns *corev1.Namespace) bool {
+	updated := false
+	if ns.Labels == nil {
+		ns.Labels = make(map[string]string)
+	}
+	for k, v := range kubeflowNamespaceLabels {
+		if _, ok := ns.Labels[k]; !ok || ns.Labels[k] != v {
```

Comment:
@hougangliu If a cluster-admin user sets a namespace label to 'disabled', it would be changed to 'enabled'. The original thinking here was to retain the same value if it's already set. Should we avoid the second condition?

Reply:
make sense. thanks